src/opcode.cc

```diff
@@ -24,17 +24,23 @@ namespace wabt {
 Opcode::Info Opcode::infos_[] = {
 #define WABT_OPCODE(rtype, type1, type2, type3, mem_size, prefix, code, Name, \
                     text, decomp)                                             \
-  {text, decomp, Type::rtype, Type::type1,                                    \
-   Type::type2, Type::type3, mem_size,                                        \
-   prefix,      code,        PrefixCode(prefix, code)},
+  {text,     decomp, Type::rtype, {Type::type1, Type::type2, Type::type3},    \
+   mem_size, prefix, code,        PrefixCode(prefix, code)},
 #include "src/opcode.def"
 #undef WABT_OPCODE
 
-  {"<invalid>", "", Type::Void, Type::Void, Type::Void, Type::Void, 0, 0, 0, 0},
+    {"<invalid>",
+     "",
+     Type::Void,
+     {Type::Void, Type::Void, Type::Void},
+     0,
+     0,
+     0,
+     0},
 };
 
 #define WABT_OPCODE(rtype, type1, type2, type3, mem_size, prefix, code, Name, \
-                    text, decomp)                                                     \
+                    text, decomp)                                             \
   /* static */ Opcode Opcode::Name##_Opcode(Opcode::Name);
```

Comment:
Changes to this file look like formatting only?   Separate PR?

Reply:
No, there are new braces around the `type1`, `type2`, `type3` now. This makes it possible to access the types by index.